pkg/util/utils.go

```diff
@@ -71,7 +69,7 @@ func Retry(attempts int, callback func() error) (err error) {
 	return RetryAfter(attempts, callback, 0)
 }
 
-func ValidateProxyURI(uri string) bool {
+func ValidProxyURI(uri string) bool {
 	return govalidator.IsURL(uri)
```

Comment:
there is no real 'default' port, as this could also be `3128` (squid). checking could introduce other issues, I understand; should it end with `/`, what if it contains a resource as `.php`/`.html`?